src/_common/filters/date.ts

```diff
@@ -10,20 +10,21 @@ import * as format from 'date-fns/format';
 // 'shortTime': 12:05 PM
 
 const ShortcutFormats = {
-	medium: 'MMM D, YYYY h:mm:ss A',
-	short: 'M/D/YY h:mm A',
-	fullDate: 'dddd, MMMM D, YYYY',
-	longDate: 'MMMM D, YYYY',
-	mediumDate: 'MMM D, YYYY',
-	shortDate: 'M/D/YY',
-	mediumTime: 'h:mm:ss A',
-	shortTime: 'h:mm A',
+	medium: 'LLL d, yyyy h:mm:ss a',
+	short: 'L/d/yy h:mm a',
+	fullDate: 'iiii, LLLL d, yyyy',
+	longDate: 'LLLL d, yyyy',
+	mediumDate: 'LLL d, yyyy',
+	shortDate: 'L/d/yy',
+	mediumTime: 'h:mm:ss a',
+	shortTime: 'h:mm a',
 };
 
 export function date(now: Date | string | number, formatTo?: string) {
 	if (formatTo && (ShortcutFormats as any)[formatTo]) {
 		formatTo = (ShortcutFormats as any)[formatTo];
 	}
 
-	return format(now, formatTo);
+	// review, format doesn't accept string anymore.
+	return format(now as number, formatTo as string);
```

Comment:
Check if we use now as a string ever intstead. Modify all cases that are passing in a string.
For the formatTo either set a default value in the function argument, or check if we have any cases where we call this without formatTo. If not, we can just make it required.

Reply:
I set `ShortcutFormats.medium` as the default for `formatTo`, just in case anyone forgets to set a format in the future.